test/models/minetype.test.js

```diff
@@ -0,0 +1,10 @@
+describe('Test minetype model', () => {
+  test('Minetype model is created', async () => {
+    jest.mock('sequelize', () => {
+      const MockSequelize = require('sequelize-mock')
+      return MockSequelize
+    })
+    const minetypeModel = require('../../server/models/minetype')
+    expect(minetypeModel.name).toEqual('')

```

Comment:
Similar to the Claim model test, I don't see what value this test provides in its current form.